### PR TITLE
Fix route that is using a string instead of account interface.

### PR DESCRIPTION
**Pull request by Jaesin**

### CODE REVIEW

File: src/Controller/BillingController.php [hunk 1]
```diff
@@ -94,23 +93,15 @@ public function myPrepaidBalance(): RedirectResponse {
...
    * @throws \Exception
    */
-  public function prepaidBalancePage(string $user) {
-    $user = User::load($user);
-
-    if (!$user) {
-      throw new NotFoundHttpException();
-    }
-
+  public function prepaidBalancePage(AccountInterface $user) {
```
Comment (dawehner) [inline src/Controller/BillingController.php]: I would have expected that you have to typehint to `UserInterface` to trigger the automatic parameter conversion.

Comment (Jaesin) [reply src/Controller/BillingController.php]: Good catch. I changed it to use `UserInterface`.

**Review by dawehner — APPROVED** [review]
Looks great!

**Comment by minnur** [comment]
Looks good

**Review by minnur — APPROVED** [review]
Looks good

**Pull request merged**